sale_automatic_workflow/tests/test_multicompany.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestMultiCompany(TransactionCase):
```

Comment:
pls use SavepointCase and move the setup to setUpClass w/ tracking disabled suite wise